packages/trail-core/README.md

```diff
@@ -8,19 +8,30 @@ trail-core is the core package. It is responsible for manipulating audit trails
 
 To install via npm:
 
-```
-npm install @nearform/trail-core
-```
+    npm install @nearform/trail-core
+
+Trail requires an instance of Postgres (version 9.5+) to function correctly. For simplicity, a preconfigured `docker-compose` file has been provided:
+
+    docker-compose up
```

Comment:
where is the docker-compose file live? They should download it from somewhere, can you put a full URL?

Reply:
It's already bundled with the module, I'll add instructions